hasher-matcher-actioner/webapp/.eslintrc.json

```diff
@@ -1,24 +1,35 @@
 {
+  "parser": "@typescript-eslint/parser",
   "env": {
     "browser": true,
     "node": true,
     "commonjs": true,
     "es6": true
   },
-  "extends": ["plugin:react/recommended", "airbnb", "prettier"],
+  "extends": [
+    "plugin:react/recommended",
+    "airbnb",
+    "prettier",
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
   "parserOptions": {
     "ecmaFeatures": {
-      "jsx": true
+      "jsx": true,
+      "tsx": true
     },
     "ecmaVersion": 12,
     "sourceType": "module"
   },
-  "plugins": ["react", "prettier"],
+  "plugins": ["react", "prettier", "@typescript-eslint"],
   "rules": {
+    "no-use-before-define": "off",
```

Comment:
The eslint error `no-use-before-define` breaks when using the `typescript-eslint` parser so we override it with `typescript-eslint`'s own version of the error. This is direct from the `typescript-eslint` documentation

Reply:
nit: link to documentation/if comments allowed provide the context here for future readers.